src/main/scala/tech/sourced/gemini/cmd/HashSparkApp.scala

```diff
@@ -23,14 +23,15 @@ case class HashAppConfig(
   sparkMemory: String = "4gb",
   sparkParallelism: Int = 8,
   docFreqFile: String = "",
-  verbose: Boolean = false
+  verbose: Boolean = false,
+  mode: String = Gemini.fileSimilarityMode
 )
 
 /**
   * Apache Spark app that applied LSH to given repos, using source{d} Engine.
   */
 object HashSparkApp extends App with Logging {
-  val repoFormats = Seq("siva", "bare", "standard")
+  val repoFormats = Seq(Gemini.fileSimilarityMode,  Gemini.funcSimilarityMode)
```

Comment:
Bug, right? `repoFormats` should not change, it's still used for `--format`, format of the stored repositories.